07-lifecycle_of_a_Patch.Rmd

```diff
@@ -112,9 +112,24 @@ When creating a patch for submission, there are several things that you can do t
 
 ## Submitting your patch for review {#SubmitPatches}
 
-1. Patch in response to a pre-existing issue or bug report:  In this case, you should attach the patch to the existing issue or bug report on Bugzilla with a brief comment.
+### Patch in response to a pre-existing issue or bug report
 
-2. Patch in response to an unreported issue or bug report: Assuming you already performed a search on Bugzilla for a pre-existing issue or bug and did not find the issue or bug reported, you need to create a new bug report and include your patch with it. Please fill in as much relevant detail as possible to prevent reviewers from having to delay reviewing your patch because of lack of information. Include (mostly as the first sentence), a to-the-point explanation of what the purpose of the patch is. This sentence should not be in the descriptive form, rather an imperative form will be more suitable here. If this is not enough detail for a patch, a new paragraphs(s) can be added to explain in proper depth what has happened. The details should be good enough that a core developer reading it understands the justification for the change.
+In this case, you should attach the patch to the existing issue or bug report on Bugzilla with a brief comment.
+
+1. Use the Attachments option to add the `*.diff` file as attachment.
+
+2. On the new Create New Attachment web page, add a Brief Decription.
+
+3. Select Content Type as `patch`.
+
+4. Add comments (often prose text) in this page rather than in the original bugzilla page for the PR.
+
+5. Press Submit. This will give you a bugzilla submission that sends one e-mail to all of R-core plus the PR author.
+
+
+### Patch in response to an unreported issue or bug report
+
+Assuming you already performed a search on Bugzilla for a pre-existing issue or bug and did not find the issue or bug reported, you need to create a new bug report and include your patch with it. Please fill in as much relevant detail as possible to prevent reviewers from having to delay reviewing your patch because of lack of information. Include (mostly as the first sentence), a to-the-point explanation of what the purpose of the patch is. This sentence should not be in the descriptive form, rather an imperative form will be more suitable here. If this is not enough detail for a patch, a new paragraphs(s) can be added to explain in proper depth what has happened. The details should be good enough that a core developer reading it understands the justification for the change.
```

Comment:
This paragraph was moved from above, but I think we can improve it too in this PR (feel free to ignore the comments):
1) Split each sentence in a different line: this will help future diffs to a single sentence.
2) The content is related to other sections of the book, I would add links to them:
  a) To search on Bugzilla (section 3.3)
  b) How to report a bug (Chapter 4)
  c) How to write a good patch (7.4)

Reconsidering, perhaps splitting this in two sections complicates following the book. Adding a sentence at the end of the "Submitting your patch for review" section highlighting the differences on this PR (without the new third level section) and if it is all in one (open bug report+patch) might be easier...
What do you think?